src/viur/core/skeleton.py

```diff
@@ -873,185 +873,179 @@ def toDB(cls, skelValues: SkeletonInstance, update_relations: bool = True, **kwa
-                    for oldValue in oldUniqueValues:
+                    for old_unique_value in old_unique_values:
                         # Try to delete the old lock
-                        oldLockKey = db.Key("%s_%s_uniquePropertyIndex" % (skel.kindName, key), oldValue)
-                        oldLockObj = db.Get(oldLockKey)
-                        if oldLockObj:
-                            if oldLockObj["references"] != dbObj.key.id_or_name:
+                        old_lock_key = db.Key(f"{skel.kindName,}_{bone_name}_uniquePropertyIndex", old_unique_value)
+                        if old_lock_obj := db.Get(old_lock_key):
+                            if old_lock_obj["references"] != db_obj.key.id_or_name:
                                 # We've been supposed to have that lock - but we don't.
                                 # Don't remove that lock as it now belongs to a different entry
                                 logging.critical("Detected Database corruption! A Value-Lock had been reassigned!")
                             else:
                                 # It's our lock which we don't need anymore
-                                db.Delete(oldLockKey)
+                                db.Delete(old_lock_key)
                         else:
                             logging.critical("Detected Database corruption! Could not delete stale lock-object!")
 
-            # Ensure the SEO-Keys are up2date
-            lastRequestedSeoKeys = dbObj["viur"].get("viurLastRequestedSeoKeys") or {}
-            lastSetSeoKeys = dbObj["viur"].get("viurCurrentSeoKeys") or {}
+            # Ensure the SEO-Keys are up-to-date
+            last_requested_seo_keys = db_obj["viur"].get("viurLastRequestedSeoKeys") or {}
+            last_set_seo_keys = db_obj["viur"].get("viurCurrentSeoKeys") or {}
             # Filter garbage serialized into this field by the SeoKeyBone
-            lastSetSeoKeys = {k: v for k, v in lastSetSeoKeys.items() if not k.startswith("_") and v}
-            currentSeoKeys = skel.getCurrentSEOKeys()
-            if not isinstance(dbObj["viur"].get("viurCurrentSeoKeys"), dict):
-                dbObj["viur"]["viurCurrentSeoKeys"] = {}
-            if currentSeoKeys:
+            last_set_seo_keys = {k: v for k, v in last_set_seo_keys.items() if not k.startswith("_") and v}
+
+            if not isinstance(db_obj["viur"].get("viurCurrentSeoKeys"), dict):
+                db_obj["viur"]["viurCurrentSeoKeys"] = {}
+            if current_seo_keys := skel.getCurrentSEOKeys():
                 # Convert to lower-case and remove certain characters
-                for lang, value in list(currentSeoKeys.items()):
-                    value = value.lower()
-                    value = value.translate(Skeleton.__seo_key_trans).strip()
-                    currentSeoKeys[lang] = value
+                for lang, value in list(current_seo_keys.items()):
```

Comment:
I think the list case is not necessary here
```suggestion
                for lang, value in current_seo_keys.items():
```